srcpkgs/atom-bin/files/atom.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
Or .desktop files.

Reply:
And what if the archive doesn't contain one?